services/catarse/dev.Dockerfile

```diff
@@ -2,17 +2,15 @@ FROM ruby:3.0.0-alpine
 #FROM alpine:3.7
 MAINTAINER Catarse <contato@catarse.me>
 
-ENV BUILD_PACKAGES postgresql-dev libxml2-dev libxslt-dev imagemagick imagemagick-dev openssl libpq libffi-dev bash curl-dev libstdc++ tzdata bash ca-certificates build-base ruby-dev libc-dev linux-headers postgresql-client postgresql git
+ENV BUILD_PACKAGES postgresql-dev libxml2-dev libxslt-dev imagemagick imagemagick-dev openssl libpq libffi-dev bash curl-dev libstdc++ tzdata bash ca-certificates build-base ruby-dev libc-dev linux-headers postgresql-client postgresql git nodejs yarn
```

Comment:
**Dúvida**
Não teria como definir a versão desse nodejs e do yarn? Pois poderemos passar pelo mesmo problema desses pacotes passarem a usar versões mais recentes sem percebermos.

E nesse arquivo talvez fosse bom lockar a versão do alpine também, o que tu acha? Ou seria melhor sempre ficar usando as mais recentes que a imagem for utilizando?

Reply:
Acho que lockar a versão do alpine ou dos pacotes. Prefiro a versão do alpine pq o alpine na versão pode dropar os pacotes (dai já fica o com o mais atualizado dos pacotes do repo deles mesmo).

Reply:
@stephannv atualizei ali pra ficar fixado a versão do alpine